README.md

```diff
@@ -1,20 +1,20 @@
-# Borg cron helper scripts with local lock system
```

Comment:
Please readd the removed part. That's the (main) feature I want to advertise with… :wink: